src/main/java/randoop/main/Minimize.java

```diff
@@ -0,0 +1,1183 @@
+      return null;
+    }
+    // Get the name of the variable being declared.
+    String varName = vdExpr.getVars().get(0).getId().getName();
+
+    // Check if the map contains a value found from a passing assertion.
+    if (primitiveValues.containsKey(varName)) {
+      String value = primitiveValues.get(varName);
+      return rhsAssignWithValue(vdExpr, vdExpr.getType(), value);
+    } else {
+      // The map does not contain a value for this variable.
+      return null;
+    }
+  }
+
+  /**
+   * Return a variable declaration statement that sets the right hand side of a variable declaration
+   * to the value that is passed in.
+   *
+   * @param vdExpr variable declaration expression representing the current statement to simplify
+   * @param value value that will be assigned to the variable being declared
+   * @return a {@code Statement} object representing the simplified variable declaration expression
+   *     if the type of the variable is a primitive and a value has been previously calculated.
+   *     Returns {@code null} if more than one variable is declared in the {@code
+   *     VariableDeclarationExpr}.
+   */
+  private static Statement rhsAssignWithValue(
+      VariableDeclarationExpr vdExpr, Type exprType, String value) {
+    if (vdExpr.getVars().size() != 1) {
+      // Number of variables declared in this expression is not 1.
+      return null;
+    }
+
+    // Create the resulting expression, a copy of the original expression
+    // which will be modified and returned.
+    VariableDeclarationExpr resultExpr = (VariableDeclarationExpr) vdExpr.clone();
+
+    // The variable declaration.
+    VariableDeclarator vd = resultExpr.getVars().get(0);
+
+    // Based on the declared variable type, set the right hand to the value
+    // that was passed in.
+    if (exprType instanceof PrimitiveType) {
+      switch (((PrimitiveType) exprType).getType()) {
+        case Boolean:
+          if (value == null) {
+            vd.setInit(new BooleanLiteralExpr(Boolean.parseBoolean("false")));
+          } else {
+            vd.setInit(new BooleanLiteralExpr(Boolean.parseBoolean(value)));
+          }
+          break;
+        case Char:
+        case Byte:
+        case Short:
+        case Int:
+          if (value == null) {
+            vd.setInit(new IntegerLiteralExpr("0"));
+          } else {
+            vd.setInit(new IntegerLiteralExpr(value));
+          }
+          break;
+        case Float:
+          if (value == null) {
+            vd.setInit(new DoubleLiteralExpr("0f"));
+          } else {
+            vd.setInit(new DoubleLiteralExpr(value));
+          }
+          break;
+        case Double:
+          if (value == null) {
+            vd.setInit(new DoubleLiteralExpr("0.0"));
+          } else {
+            vd.setInit(new DoubleLiteralExpr(value));
+          }
+          break;
+        case Long:
+          if (value == null) {
+            vd.setInit(new LongLiteralExpr("0l"));
```

Comment:
Always use uppercase "L" in long literal expressions.  The lowercase letter "l" looks too much like the number "1".